### PR TITLE
[202205][PTF python3 migration] Modify test_voq_nbr and test_voq_ipfwd PTF script

### PR DESCRIPTION
**Pull request by wsycqyz**
### Description of PR

Summary:
Manually cherry-pick #9449
Migrate test_voq_nbr and test_voq_ipfwd PTF script to Python3

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

### CODE REVIEW

File: ansible/roles/test/files/ptftests/py3/voq.py [hunk 1]
```diff
@@ -249,14 +249,14 @@ def runTest(self):
         send_pkt, masked_pkt, exp_pkt255 = build_ttl0_pkts(version, dst_mac, dst_ip, vm_mac, vm_ip, dut_lb)
         logger.info("sending packet to port %s", src_port)
         send(self, src_port, send_pkt)
-        print("masked packet matched port: %s" % src_port)
+        print(("masked packet matched port: %s" % src_port))
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/py3/voq.py:252]: Extra `()` here.

File: ansible/roles/test/files/ptftests/py3/voq.py [hunk 2]
```diff
@@ -249,14 +249,14 @@ def runTest(self):
...
 
         result = dp_poll(self, device_number=0, exp_pkt=masked_pkt, timeout=2)
         self.at_receive(result.packet, device_number=result.device, port_number=result.port)
 
-        print("Found %s ICMP ttl expired packets on ports: %s" % (result, str(src_rx_ports)))
+        print(("Found %s ICMP ttl expired packets on ports: %s" % (result, str(src_rx_ports))))
         logger.info("Found %s ICMP ttl expired packets on ports: %s" % (result, str(src_rx_ports)))
-        print("port: %s" % result.port)
+        print(("port: %s" % result.port))
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/py3/voq.py:259]: Extra `()` here.

File: ansible/roles/test/files/ptftests/py3/voq.py [hunk 3]
```diff
@@ -249,14 +249,14 @@ def runTest(self):
...
         logger.info("sending packet to port %s", src_port)
         send(self, src_port, send_pkt)
-        print("masked packet matched port: %s" % src_port)
+        print(("masked packet matched port: %s" % src_port))
 
         result = dp_poll(self, device_number=0, exp_pkt=masked_pkt, timeout=2)
         self.at_receive(result.packet, device_number=result.device, port_number=result.port)
 
-        print("Found %s ICMP ttl expired packets on ports: %s" % (result, str(src_rx_ports)))
+        print(("Found %s ICMP ttl expired packets on ports: %s" % (result, str(src_rx_ports))))
```
[the remark this answers is not in the document]
Comment (wsycqyz) [reply ansible/roles/test/files/ptftests/py3/voq.py]: fixed.

File: ansible/roles/test/files/ptftests/py3/voq.py (hunk 1 above)
Comment (wsycqyz) [reply ansible/roles/test/files/ptftests/py3/voq.py]: fixed.

File: ansible/roles/test/files/ptftests/py3/voq.py (hunk 2 above)
Comment (wsycqyz) [reply ansible/roles/test/files/ptftests/py3/voq.py]: fixed.

**Pull request merged**